### PR TITLE
Add table block to specification

### PR DESCRIPTION
**Pull request by nlisgo**
@thewilkybarkid

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Looks good; you'll need to execute `node compile.js` and commit the results.

File: src/blocks/table.v1.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Table
+type: object
+properties:
+    type:
+        type: string
+        enum:
+          - table
+    html:
+        title: HTML
```
Comment (thewilkybarkid) [inline src/blocks/table.v1.yaml]: Might be worth expanding on this. 'HTML table'?

**Comment by thewilkybarkid** [comment]
@nlisgo Following #8 could you `npm install` then `node compile.js`. This should remove `dist/scripts/api-console-vendor.js` from being included.

**Comment by nlisgo** [comment]
@thewilkybarkid I rebased and confirm that my PR no longer includes changes to `dist/scripts/api-console-vendor.js`

**Pull request merged**